src/subscribr.js

```diff
@@ -55,6 +55,12 @@ module.exports = class Subscribr {
 
 }
 
+let idCounter = 0;
```

Comment:
That looks better. I guess it could me moved to a different module, since it is a completely different functionality, or it could be used the uniqueId feature of lodash (didn't add it because i didn't know your policy about adding libraries).

What would you prefer?